butterknife/build.gradle

```diff
@@ -1,4 +1,5 @@
 apply plugin: 'com.android.library'
+apply plugin: 'com.kageiit.lintrules'
```

Comment:
Ya that part is for apps, which we are not using here. I am planning to remove custom lint for app projects from the plugin and cut a new release soon.